ec2/spark_ec2.py

```diff
@@ -1007,6 +1023,11 @@ def real_main():
         print >> stderr, "ebs-vol-num cannot be greater than 8"
         sys.exit(1)
 
+    # Prevent breaking ami_prefix
```

Comment:
If we're validating this input, perhaps we should also check that the repo string starts with "https://github.com". Sounds good?